<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParameterDialog</class>
 <widget class="QDialog" name="ParameterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактирование параметра</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/resources.qrc">
    <normaloff>:/resources/icons/edit.png</normaloff>:/resources/icons/edit.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="parameterGroup">
     <property name="title">
      <string>Параметр</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="paramNameLabel">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Название параметра:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="parameterNameEdit">
        <property name="toolTip">
         <string>Введите название параметра из СОТМ</string>
        </property>
        <property name="placeholderText">
         <string>Например: НОВФР</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="conditionLabel">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Метод контроля:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="methodGroup">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="equalsLayout">
        <item>
         <widget class="QRadioButton" name="equalsRadio">
          <property name="text">
           <string>Равен:</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="equalsValueEdit">
          <property name="placeholderText">
           <string>Значение</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="notEqualsLayout">
        <item>
         <widget class="QRadioButton" name="notEqualsRadio">
          <property name="text">
           <string>Не равен:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="notEqualsValueEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="placeholderText">
           <string>Значение</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="inLimitsRadio">
        <property name="text">
         <string>Лежит в пределах:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="inLimitsLayout">
        <item>
         <widget class="QLineEdit" name="lowerLimitEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="placeholderText">
           <string>Нижний предел</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelLowerLimit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>&lt;= значение &lt;=</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="upperLimitEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="placeholderText">
           <string>Верхний предел</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="outOfLimitsRadio">
        <property name="text">
         <string>Запрещенный диапазон:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="outOfLimitsLayout">
        <item>
         <widget class="QLabel" name="labelLowerOutLimit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>от</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lowerOutLimitEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="placeholderText">
           <string>Нижний предел</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelUpperOutLimit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>до</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="upperOutLimitEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="placeholderText">
           <string>Верхний предел</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="changedRadio">
        <property name="text">
         <string>Фиксация каждого изменения</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="descriptionGroup">
     <property name="title">
      <string>Описание параметра</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QTextEdit" name="descriptionEdit">
        <property name="placeholderText">
         <string>Введите описание параметра, условий контроля, ссылки на документацию...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="soundGroupBox">
     <property name="title">
      <string>Звуковое оповещение</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="soundFileEdit">
        <property name="placeholderText">
         <string>Путь к звуковому файлу (.wav)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="soundFileBrowseButton">
        <property name="text">
         <string>Выбрать</string>
        </property>
        <property name="icon">
         <iconset resource="../resources/resources.qrc">
          <normaloff>:/resources/icons/sound_on.png</normaloff>:/resources/icons/sound_on.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
     <layout class="QHBoxLayout" name="enableSoundLayout">
      <item>
       <widget class="QCheckBox" name="enableSoundCheckBox">
        <property name="text">
         <string>Включить звуковое оповещение</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ParameterDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ParameterDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>